components/form/Select.js

```diff
@@ -0,0 +1,69 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Label, ErrorMessage } from '.';
+import { useFormContext } from 'react-hook-form';
+
+export function SelectField({ className, containerClassName, label, labelClassName, name, ...options }) {
+  const {
+    register,
+    formState: { errors }
+  } = useFormContext();
+
+  const hasErrors = !!errors?.[name];
+
+  const errorClass = hasErrors ? 'focus:border-amcovad-danger' : 'focus:border-amcovad-primary-500';
+
+  const style = {
+    container: `relative my-1 `,
+    default: `${errorClass} relative flex flex-1 w-full mt-1 rounded-md py-2 px-2 bg-transparent text-amcovad-secondary-700 placeholder-amcovad-secondary-700 border border-amcovad-secondary-300 text-base focus:outline-none focus:ring-1 peer border-2 `
+  };
+
+  return (
+    <div className={`${style.container} ${containerClassName}`}>
+      <Label htmlFor={name} name={name} text={label} className={`!text-sm ${labelClassName}`} />
```

Comment:
The exclamation mark seems like a typo to me so we can remove it.

```suggestion
      <Label htmlFor={name} name={name} text={label} className={`text-sm ${labelClassName}`} />
```

Reply:
it's `! important ` the default label font is `text-xs = 12px` here changing it to `text-sm =14px` to over it